common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
 function (VideoPlayer, i18n) {
     var moment = window.moment;
```

Comment:
I'm also confused as to whether or not we should have the following someplace, per http://momentjs.com/docs/#/use-it/require-js/

define(["moment"], function (moment) {
    moment().format();
});